src/App.tsx

```diff
@@ -1,8 +1,42 @@
 import React from 'react';
+import {ReactComponent as btnIcon} from 'assets/button-icon.svg';
+import Button from 'components/atoms/Button';
+import { EIconPosition } from 'components/atoms/Button/Button.model';
```

Comment:
In each case of use this button, should we import this enum? I see that enum are problematic

Reply:
I think, if in specific case we have only 2-3 specific strings, we can set it in interface e.q:  
position: 'left' | 'right' | 'center';  
TS will do rest ;)